recOrder/scripts/cli.py

```diff
@@ -1,6 +1,9 @@
 import click
 import napari
+
+v = napari.Viewer()  # open viewer right away to use on hpc
```

Comment:
this earlier issue https://github.com/mehta-lab/recOrder/issues/315 popped up again, so I needed to create the viewer at the top